Project 1/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
Before you merge your project, you should delete this file and rename your short story HTML to `index.html` 👍🏻